monitoring/daily_report.py

```
@@ -0,0 +1,654 @@
+- SENDGRID_FROM_EMAIL: Sender email address
+"""
+
+import os
+import sys
+import json
+import gzip
+import re
+import logging
+from datetime import datetime, timezone, timedelta
+from pathlib import Path
+from typing import Dict, Any, List, Optional, Set
```

Ruff (0.14.14)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)